src/test/java/com/github/freeclimbapi/RequestVerifierTest.java

```diff
@@ -0,0 +1,119 @@
+        String signingSecret = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+        String requestHeader = "t=1679944186,v1=c3957749baf61df4b1506802579cc69a74c77a1ae21447b930e5a704f9ec4120,v1=1ba18712726898fbbe48cd862dd096a709f7ad761a5bab14bda9ac24d963a6a8";
+        int tolerance = Integer.MAX_VALUE;
+        instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void checkToleranceTest2() {
+        String requestBody = "{\"accountId\":\"AC1334ffb694cd8d969f51cddf5f7c9b478546d50c\",\"callId\":\"CAccb0b00506553cda09b51c5477f672a49e0b2213\",\"callStatus\":\"ringing\",\"conferenceId\":null,\"direction\":\"inbound\",\"from\":\"+13121000109\",\"parentCallId\":null,\"queueId\":null,\"requestType\":\"inboundCall\",\"to\":\"+13121000096\"}";
+        String signingSecret = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+        String requestHeader = "t=1679944186,v1=c3957749baf61df4b1506802579cc69a74c77a1ae21447b930e5a704f9ec4120,v1=1ba18712726898fbbe48cd862dd096a709f7ad761a5bab14bda9ac24d963a6a8";
+        int tolerance = -5;
+        instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void checkToleranceTest3() {
+        String requestBody = "{\"accountId\":\"AC1334ffb694cd8d969f51cddf5f7c9b478546d50c\",\"callId\":\"CAccb0b00506553cda09b51c5477f672a49e0b2213\",\"callStatus\":\"ringing\",\"conferenceId\":null,\"direction\":\"inbound\",\"from\":\"+13121000109\",\"parentCallId\":null,\"queueId\":null,\"requestType\":\"inboundCall\",\"to\":\"+13121000096\"}";
+        String signingSecret = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+        String requestHeader = "t=1679944186,v1=c3957749baf61df4b1506802579cc69a74c77a1ae21447b930e5a704f9ec4120,v1=1ba18712726898fbbe48cd862dd096a709f7ad761a5bab14bda9ac24d963a6a8";
+        int tolerance = 0;
+        instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void verifyToleranceTest() {
+        String requestBody = "{\"accountId\":\"AC1334ffb694cd8d969f51cddf5f7c9b478546d50c\",\"callId\":\"CAccb0b00506553cda09b51c5477f672a49e0b2213\",\"callStatus\":\"ringing\",\"conferenceId\":null,\"direction\":\"inbound\",\"from\":\"+13121000109\",\"parentCallId\":null,\"queueId\":null,\"requestType\":\"inboundCall\",\"to\":\"+13121000096\"}";
+        String signingSecret = "sigsec_ead6d3b6904196c60835d039e91b3341c77a7793";
+        String requestHeader = "t=1900871395,v1=1d798c86e977ff734dec3a8b8d67fe8621dcc1df46ef4212e0bfe2e122b01bfd,v1=1ba18712726898fbbe48cd862dd096a709f7ad761a5bab14bda9ac24d963a6a8";
+        int tolerance = 5 * 60;
+        instance.verifyRequestSignature(requestBody, requestHeader, signingSecret, tolerance);
+    }
+
+    @Test(expected = RuntimeException.class)
```

Comment:
All of the tests expecting exceptions are not very specific since they all use `RuntimeException`. How can we be more testing which exception is thrown?

Reply:
According to this stackoverflow link, i can check the message to make it more specific, that way we will know for sure that this specific error message is thrown for the runtime exception

Reply:
There is this stackoverflow link I found where we can specify the error message coming back:
https://stackoverflow.com/a/52085072

I can refactor the tests to use this method in the latest commit